Framework/PythonInterface/plugins/functions/ChudleyElliot.py

```diff
@@ -44,7 +48,7 @@ def function1D(self, xvals):
         length = self.getParameterValue("L")
 
         xvals = np.array(xvals)
-        hwhm = (1.0 - np.sin(xvals * length) / (xvals * length)) / tau
+        hwhm = self.hbar*(1.0 - np.sin(xvals * length) / (xvals * length))/tau
```

Comment:
There is some mixing of whitespace conventions in this equation. Should the operators / and * have either surrounding whitespace or no surrounding whitespace, but not both? There could be good reason for this and I am mistaken!